distributionviewer/core/static/js/app/components/containers/chart-list-container.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ChartList from '../views/chart-list';
+
+// numCharts is updated and provided to ChartList just for demonstration
+// purposes, and so that ChartList doesn't just have to include <ExampleChart />
+// 30 times in its source.
+const ChartListContainer = React.createClass({
+  getInitialState: function() {
+    return {
+      numCharts: 0
+    }
+  },
+
+  componentDidMount: function() {
```

Comment:
Ah, agreed. I was even thinking of moving us to OO-based React code, which should be easier to read.